### DISCUSSION TITLE
Where is @swc/jest version 0.2.24 ?

### DISCUSSION THREAD

**Discussion by tjunghans** [swc-project/swc#6709] [Q&A] [answered]
I installed [@swc/jest@0.2.24](https://www.npmjs.com/package/@swc/jest/v/0.2.24?activeTab=explore) yesterday and several tests started failing. I attempted to look for the diff between 02.24 and 0.2.23 (which works) but I cannot find it on Github. ~~The latest version on [master is 0.2.23](https://github.com/swc-project/jest/blob/master/package.json).~~ 

Before I file a bug I'd like to see what changed. It looks very much like the update and npm publish were done locally without Git but perhaps I missed something?

For the curious, the below error appears in several test suites ~~with @swc/jest@0.2.24.~~ **after updating @swc/core from 1.2.203 to 1.3.24**.

Test suite failed to run

    TypeError: Cannot read properties of undefined (reading 'endCol')

      at module.exports.sliceRange (node_modules/v8-to-istanbul/lib/range.js:30:54)
      at node_modules/v8-to-istanbul/lib/v8-to-istanbul.js:139:19
          at Array.forEach (<anonymous>)
      at node_modules/v8-to-istanbul/lib/v8-to-istanbul.js:125:20
          at Array.forEach (<anonymous>)
      at V8ToIstanbul.applyCoverage (node_modules/v8-to-istanbul/lib/v8-to-istanbul.js:124:12)
      at node_modules/@jest/reporters/build/CoverageReporter.js:530:21
          at async Promise.all (index 21)
      at CoverageReporter._getCoverageResult (node_modules/@jest/reporters/build/CoverageReporter.js:499:35)
      at CoverageReporter.onRunComplete (node_modules/@jest/reporters/build/CoverageReporter.js:172:34)
      at ReporterDispatcher.onRunComplete (node_modules/@jest/core/build/ReporterDispatcher.js:64:9)
      at TestScheduler.scheduleTests (node_modules/@jest/core/build/TestScheduler.js:296:5)
      at runJest (node_modules/@jest/core/build/runJest.js:349:19)

**Comment by kwonoj** [comment]
1. this should be reported to https://github.com/swc-project/jest
2. you can actually compare changes, npm's code tab you linked is versioned https://www.npmjs.com/package/@swc/jest/v/0.2.24?activeTab=explore / https://www.npmjs.com/package/@swc/jest/v/0.2.23?activeTab=explore
3. The only difference between 2 version is https://github.com/swc-project/jest/commit/08419c0d91b4952e50b519cf378ada85636b0be9 . it doesn't look like this change could cause your error.

**Reply by kwonoj** [reply]
I'll transfer issue soon.

**Comment by tjunghans** [comment]
Argh!. Of course I only notice this after I post a question. The issue I have is not related to @swc/jest. The question of the missing source code in Github still stands, but the stack trace I posted further up is actually the result of updating @swc/core from 1.2.203 to 1.3.24. I will investigate further before posting an issue.

**Reply by kwonoj** [reply]
> The question of the missing source code in Github still stands, 

That still belongs to swc/jest repo.

**Reply by tjunghans** [reply]
Noted. The thing is, I cannot. Issues seem to be disabled in swc/jest.

**Comment by kdy1** [comment, marked as answer]
Fixed by https://github.com/swc-project/jest/commit/b77510467470eefd908aa245e9e8ad462b8fbb77
